File: src/main/js/WeatherForecast.js
'use strict';

import React from 'react';
import moment from 'moment';

export default ({title, weatherForecast, backgroundColor}) => {
    return (
        <div><h2 style={{backgroundColor: backgroundColor}}>{title}</h2>
            <div>
                <table>
                    <tbody>
                    <tr>
                        <td>CityName:</td>
                        <td>{weatherForecast.cityName}</td>
                    </tr>
                    <tr>
                        <td>Temperature:</td>
                        <td>{weatherForecast.temperature}°C</td>
                    </tr>
                    <tr>
                        <td>Humidity:</td>
                        <td>{weatherForecast.humidity}%</td>
                    </tr>
                    <tr>
                        <td>Wind:</td>
                        <td>{weatherForecast.wind}km/h</td>
                    </tr>
                    <tr>
                        <td>Precipitation:</td>
                        <td>{weatherForecast.precipitation}%</td>
                    </tr>
                    <tr>
                        <td>Forecast Time:</td>
                        <td>{moment(weatherForecast.forecastTime).format("DD/MM/YYYY HH:mm:ss")}</td>
                    </tr>
                    </tbody>

                </table>
            </div>
        </div>
    )
}

File: src/main/java/com/pointr/domain/WeatherForecast.java
package com.pointr.domain;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@ToString
@Entity
public class WeatherForecast {

    @Id
    @GeneratedValue
    private Integer id;

    private Integer cityId;

    private String cityName;

    private LocalDateTime forecastTime;

    private Double temperature;

    private Double humidity;

    private Double wind;

    private Double precipitation;

}
